docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -3,16 +3,48 @@ title: Deploying Azure Service Operator v2 in multi-tenant mode
 linktitle: Multitenancy
 ---
 
-The current release version of the Azure Service Operator (ASO) deployment YAML installs the operator in _single-tenant_ mode:
-deployed in the `azureserviceoperator-system` namespace with _a single set of Azure credentials_,
-and manages resources in _any namespace_ in the cluster.
-That single operator deployment handles webhooks fired when Azure resources are changed.
+Currently, we support two types of multitenancy with Azure Service Operator (ASO): 
+
+### Single operator multitenancy (default)
```

Comment:
Heading indentation levels don't really make sense here. These headings should be able the same level as the "XYZ deployment" ones below, so 2 `#`'s not 3.

Or else we could do:

```
## Single Operator multitenancy (default)
...

### Deployment
...

## Multiple operator multitenancy
...

### Deployment
...
```